packages/cursorless-engine/src/runCommand.ts

```diff
@@ -35,6 +40,7 @@ export async function runCommand(
   languageDefinitions: LanguageDefinitions,
   rangeUpdater: RangeUpdater,
   commandRunnerDecorators: CommandRunnerDecorator[],
+  commandHistory: CommandHistory,
```

Comment:
Why can't this be a command decorator? (See the commandDecorators arg in the previous line (line 42))

Reply:
No reason I just wanted a stricter typing. Probably unnecessary and I have re implemented it as a command runner decorator instead.